packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -288,28 +338,44 @@ export class OrderedClientElection
                 });
```

Comment:
The "if" block basically checks whether the elected client as per the initial state in summary exists. If not, `this._electedClient` will be undefined. I have couple of questions here:
1. Does this trigger a re-election?
2. The initial state in summary may be out of date right? Basically, there may already be a new summarizer chosen.

Reply:
1. I don't think a re-election will be triggered immediately, but if we process an op, and _electedClient is undefined, then we'll do a new election, so it will be triggered soon.
2. Yes, you're right. In the case where we went too many ops without summarizing and elected a new one, but the new one hasn't completed a summary yet at the time the new client is loaded, we could read the old (unresponsive) client's ID from the snapshot we use to load the new client. If the unresponsive client is still in the quorum, the new client will think the unresponsive client is the elected summarizer. I _think_ this is only a problem when the new client becomes the elected client but doesn't know it. I'm not sure that's possible.

Reply:
Thanks, that makes sense. This is really a corner case and should not happen often. I was worried if this can happen a lot and looks like that's not the case.